src/main/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlight.java

```diff
@@ -248,6 +252,29 @@ public SnapshotCreateFlight(FlightMap inputParameters, Object applicationContext
                   azureSynapsePdao, snapshotService, snapshotReq));
         }
         break;
+      case BYREQUESTID:
+        if (platform.isGcp()) {
+          addStep(
+              new CreateSnapshotByRequestIdGcpStep(
+                  snapshotReq,
+                  snapshotService,
+                  snapshotBuilderService,
+                  snapshotRequestDao,
+                  snapshotDao,
+                  userReq,
+                  bigQuerySnapshotPdao));
+        } else if (platform.isAzure()) {
+          addStep(
+              new CreateSnapshotByRequestIdAzureStep(
+                  snapshotReq,
+                  snapshotBuilderService,
+                  snapshotService,
```

Comment:
The azure and gcp have the order of these arguments swapped, is there a reason for this difference?